packages/kbn-test/README.md

```diff
@@ -0,0 +1,80 @@
+Kibana Testing Library
+======================
+
+The @kbn/test package provides ways to run tests. Currently only integration testing is provided by this library, with unit and other testing possibly added here.
+
+Integration Testing
+-------------------
+
+### Dependencies
+
+Integration testing methods exist in the `src/functional_tests` directory. They depend on the Functional Test Runner, which is found in [`{KIBANA_ROOT}/src/functional_test_runner`](../../src/functional_test_runner). Ideally libraries provided by kibana packages such as this one should not depend on kibana source code that lives in [`{KIBANA_ROOT}/src`](../../src). The goal is to start pulling test and development utilities out into packages so they can be used across kibana and x-pack-kibana. Accordingly the Functional Test Runner itself will be pulled out into a package (or part of a package), and this package's dependence on it will not be an issue.
```

Comment:
Maybe i should say `Kibana and X-Pack Kibana` to avoid naming confusion.

Reply:
We plan to merge the x-pack test stuff (at least form a CLI perspective) into the top level test commands over time. It'll be like the build stuff exists now, where you pass an `--oss` argument to the test CLI to ignore x-pack tests.

Given that, perhaps the language here should focus more broadly on "Kibana and plugins" rather than "Kibana and X-Pack".